boards.txt

```diff
@@ -26494,6 +26494,229 @@ fm-devkit.menu.EraseFlash.none.upload.erase_cmd=
+fri3d_2024_esp32s3.build.memory_type={build.boot}_{build.psram_type}
+
+## IDE 2.0 Seems to not update the value
+fri3d_2024_esp32s3.menu.JTAGAdapter.default=Disabled
+fri3d_2024_esp32s3.menu.JTAGAdapter.default.build.copy_jtag_files=0
+fri3d_2024_esp32s3.menu.JTAGAdapter.builtin=Integrated USB JTAG
+fri3d_2024_esp32s3.menu.JTAGAdapter.builtin.build.openocdscript=esp32s3-builtin.cfg
+fri3d_2024_esp32s3.menu.JTAGAdapter.builtin.build.copy_jtag_files=1
+fri3d_2024_esp32s3.menu.JTAGAdapter.external=FTDI Adapter
+fri3d_2024_esp32s3.menu.JTAGAdapter.external.build.openocdscript=esp32s3-ftdi.cfg
+fri3d_2024_esp32s3.menu.JTAGAdapter.external.build.copy_jtag_files=1
+fri3d_2024_esp32s3.menu.JTAGAdapter.bridge=ESP USB Bridge
+fri3d_2024_esp32s3.menu.JTAGAdapter.bridge.build.openocdscript=esp32s3-bridge.cfg
+fri3d_2024_esp32s3.menu.JTAGAdapter.bridge.build.copy_jtag_files=1
+
+fri3d_2024_esp32s3.menu.PSRAM.default=OPI PSRAM
+fri3d_2024_esp32s3.menu.PSRAM.default.build.defines=-DBOARD_HAS_PSRAM
+fri3d_2024_esp32s3.menu.PSRAM.default.build.psram_type=opi
+fri3d_2024_esp32s3.menu.PSRAM.disabled=Disabled
+fri3d_2024_esp32s3.menu.PSRAM.disabled.build.defines=
+fri3d_2024_esp32s3.menu.PSRAM.disabled.build.psram_type=qspi
+fri3d_2024_esp32s3.menu.PSRAM.enabled=QSPI PSRAM
+fri3d_2024_esp32s3.menu.PSRAM.enabled.build.defines=-DBOARD_HAS_PSRAM
+fri3d_2024_esp32s3.menu.PSRAM.enabled.build.psram_type=qspi
+
+
+fri3d_2024_esp32s3.menu.FlashMode.qio=QIO 80MHz
+fri3d_2024_esp32s3.menu.FlashMode.qio.build.flash_mode=dio
+fri3d_2024_esp32s3.menu.FlashMode.qio.build.boot=qio
+fri3d_2024_esp32s3.menu.FlashMode.qio.build.boot_freq=80m
+fri3d_2024_esp32s3.menu.FlashMode.qio.build.flash_freq=80m
+fri3d_2024_esp32s3.menu.FlashMode.qio120=QIO 120MHz
+fri3d_2024_esp32s3.menu.FlashMode.qio120.build.flash_mode=dio
+fri3d_2024_esp32s3.menu.FlashMode.qio120.build.boot=qio
+fri3d_2024_esp32s3.menu.FlashMode.qio120.build.boot_freq=120m
+fri3d_2024_esp32s3.menu.FlashMode.qio120.build.flash_freq=80m
+fri3d_2024_esp32s3.menu.FlashMode.dio=DIO 80MHz
+fri3d_2024_esp32s3.menu.FlashMode.dio.build.flash_mode=dio
+fri3d_2024_esp32s3.menu.FlashMode.dio.build.boot=dio
+fri3d_2024_esp32s3.menu.FlashMode.dio.build.boot_freq=80m
+fri3d_2024_esp32s3.menu.FlashMode.dio.build.flash_freq=80m
+fri3d_2024_esp32s3.menu.FlashMode.opi=OPI 80MHz
+fri3d_2024_esp32s3.menu.FlashMode.opi.build.flash_mode=dout
+fri3d_2024_esp32s3.menu.FlashMode.opi.build.boot=opi
+fri3d_2024_esp32s3.menu.FlashMode.opi.build.boot_freq=80m
+fri3d_2024_esp32s3.menu.FlashMode.opi.build.flash_freq=80m
+
+fri3d_2024_esp32s3.menu.FlashSize.default=16MB (128Mb)
+fri3d_2024_esp32s3.menu.FlashSize.default.build.flash_size=16MB
+fri3d_2024_esp32s3.menu.FlashSize.4M=4MB (32Mb)
+fri3d_2024_esp32s3.menu.FlashSize.4M.build.flash_size=4MB
+fri3d_2024_esp32s3.menu.FlashSize.8M=8MB (64Mb)
+fri3d_2024_esp32s3.menu.FlashSize.8M.build.flash_size=8MB
+fri3d_2024_esp32s3.menu.FlashSize.8M.build.partitions=default_8MB
```

Comment:
Can be probably removed, as your badge is using the `ESP32-S3-WROOM-1-N16R8` with 16MB flash.
```suggestion
```